polymer.json

```diff
@@ -21,7 +21,8 @@
     "bower.json",
     "manifest.json",
     "robots.txt",
-    "sitemap.xml"
+    "sitemap.xml",
+    ".well-known/**"
```

Comment:
Looks like the list is alphaordered. Generally `.` is before all letters. I would put it at the beginning.